.github/workflows/docs-push.yml

```diff
@@ -34,8 +34,9 @@ jobs:
       - name: Copy files
         run: |
           # Specify the files or directories you want to copy
-          cp -r docs/* knowledge-base/developer-documentation/wallet-sdk/
-          cp README.md knowledge-base/developer-documentation/wallet-sdk/README.md
+          rm -rf ./knowledge-base/credential-wallet/wallet-sdk
+          rsync -av docs/ ./knowledge-base/credential-wallet/wallet-sdk/
```

Comment:
If you need to exclude the API docs from the deletion, then I recommend using rsync --delete --exclude . Then you don't need a separate rm command.
If you don't need to exclude the API docs from the deletion, then you can use rm and cp and you don't need rsync.